Classes/TMDebugConsoleMessage.m

```diff
@@ -0,0 +1,25 @@
+//
+//  TMDebugConsoleMessage.m
+//  Messenger
+//
+//  Created by cestebanez on 07/09/15.
+//  Copyright (c) 2015 Tuenti Technologies S.L. All rights reserved.
+//
+
+#import "TMDebugConsoleMessage.h"
+
+@implementation TMDebugConsoleMessage
+
+- (instancetype)initWithLogMessage:(NSString *)logMessage timestamp:(NSDate *)timestamp logFlag:(int)logFlag
+{
+	self = [super init];
+	if (self)
+	{
+		_logMessage = [logMessage copy];
```

Comment:
Let's be safe in this case, otherwise, this whole TMDebugConsoleMessage class doesn't make much sense, does it?